project-koku/koku
# [COST-3039] Move ProviderInfrastructureMap off of get_or_create in provider_db_accessor

## Jira Ticket

[COST-3039](https://issues.redhat.com/browse/COST-3039)

## Description

This change will switch from using `get_or_create` to filtering for the first entry that matches and using that if it exists or creating an entry if it does not for `ProviderInfrastructureMap` objects.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load in OCP on cloud data such as with: `make create-test-customer` followed by `make load-test-customer-data`
4. See that data still processes normally and all is well.

## Notes

...
